src/index.ml

```diff
@@ -241,14 +216,14 @@ struct
         IO.v ~fresh:false ~readonly:true ~generation:0L ~fan_size:0L path
       in
       let mem = Tbl.create 0 in
-      iter_io (fun e -> Tbl.replace mem e.key e.value) io;
+      IO.iter (fun _ e -> Tbl.replace mem e.key e.value) io;
```

Comment:
Maybe you can add an `iter_io` function to make the diff a bit smaller.

Reply:
Yep, thanks.